tests/Nelmio/Alice/support/fixtures/unique_with_template.yml

```diff
@@ -0,0 +1,6 @@
+\Nelmio\Alice\support\models\User:
```

Comment:
I think we need a more complete test for that one, e.g. add another non unique field to the template. It would be also good to have another example with two level deep, i.e.

- template1: 2 fields with unique, 1 without
- template2: extends template 1, override 1 unique field to make it not unique, add another unique field
- 3: extends template 1

Don't hesitate to create dedicated entities for it if needed